pkg/kubelet/eviction/helpers.go

```diff
@@ -364,6 +366,16 @@ func parseThresholdStatements(statements map[string]string) ([]evictionapi.Thres
 			results = append(results, *result)
 		}
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.InheritDefaultEvictionValues) {
+		for signal, val := range DefaultEvictionHard {
```

Comment:
parseThresholdStatements is used for hard and soft thresholds... does including hard defaults for both make sense?

Reply:
Thank you for this point. So, should I update this change outside the parseThresholdStatements ?